### PR TITLE
fix(global support): made first round of edits to this support doc

### PR DESCRIPTION
**Pull request by homelessbirds**
Made edits for a release on May 9th.

### CODE REVIEW

**Review by akristen — DISMISSED** [review]
Looks good to me!!! didn't see any typos and it rendered correctly on my preview :)

File: src/content/docs/licenses/license-information/general-usage-licenses/global-technical-support-offerings.mdx [hunk 1]
```diff
@@ -234,14 +202,116 @@ These Support Plans apply only to your paid service subscription under an existi
           </td>
         </tr>
       </tbody>
-    </table>
+</table>
+
+### Response times 
+
+The table below shows estimated response times based on your New Relic usage plan.
```
Comment (rhetoric101) [inline src/content/docs/licenses/license-information/general-usage-licenses/global-technical-support-offerings.mdx:209]: @homelessbirds I think Standard, Pro, and Enterprise are called "Editions"--not usage plans. See this doc: https://docs.newrelic.com/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing/

File: src/content/docs/licenses/license-information/general-usage-licenses/global-technical-support-offerings.mdx [hunk 2]
```diff
@@ -234,14 +202,116 @@ These Support Plans apply only to your paid service subscription under an existi
...
+                All other issues for an existing monitored application
+            </td>
+            <td>
+                2  business days
+            </td>
+            <td>
+                1 business day
+            </td>
+            <td>
+                1 business days
```
Comment (rhetoric101) [inline src/content/docs/licenses/license-information/general-usage-licenses/global-technical-support-offerings.mdx:286]:
```suggestion
                1 business day
```

File: src/content/docs/licenses/license-information/general-usage-licenses/global-technical-support-offerings.mdx [hunk 3]
```diff
@@ -234,14 +202,116 @@ These Support Plans apply only to your paid service subscription under an existi
...
+        </tr>
+        <tr>
+            <td>
+                P3
+            </td>
+            <td>
+                All other issues for an existing monitored application
+            </td>
+            <td>
+                2  business days
```
Comment (rhetoric101) [inline src/content/docs/licenses/license-information/general-usage-licenses/global-technical-support-offerings.mdx:280]:
```suggestion
                2 business days
```

File: src/content/docs/licenses/license-information/general-usage-licenses/global-technical-support-offerings.mdx [hunk 4]
```diff
@@ -234,14 +202,116 @@ These Support Plans apply only to your paid service subscription under an existi
...
+        <tr>
+            <td>
+                P1
+            </td>
+            <td>
+                * A majority of New Relic platform functionality is down or instrumentation is not reporting data. 
+                * Observability is completely unavailable for an existing business-critical application. No workaround is available. 
+            </td>
+            <td>
+                2 business days
```
Comment (rhetoric101) [inline src/content/docs/licenses/license-information/general-usage-licenses/global-technical-support-offerings.mdx:245]: @homelessbirds I wonder if we need to follow the style guide here and spell out the days under 10. What do you think? For example, "Two hours," etc..

File: src/content/docs/licenses/license-information/general-usage-licenses/global-technical-support-offerings.mdx [hunk 5]
```diff
@@ -234,14 +202,116 @@ These Support Plans apply only to your paid service subscription under an existi
...
+            </td>
+        </tr>
+    </tbody>
+</table>
 
     **Notes:**
 
     * If you have not upgraded or changed to the [New Relic usage-based pricing model](/docs/accounts/accounts-billing/new-relic-one-pricing-users/pricing-billing), your existing support plan still applies.
```
Comment (rhetoric101) [inline src/content/docs/licenses/license-information/general-usage-licenses/global-technical-support-offerings.mdx:311]: Should we perhaps link to the next collapser for details about that older plan?

File: src/content/docs/licenses/license-information/general-usage-licenses/global-technical-support-offerings.mdx [hunk 6]
```diff
@@ -574,17 +596,106 @@ These Support Plans apply only to your paid service subscription under an existi
...
+                P3
+            </td>
+            <td>
+                1 business day
+            </td>
+            <td>
+                1 business day
+            </td>
+            <td>
+                1 business days
```
Comment (rhetoric101) [inline src/content/docs/licenses/license-information/general-usage-licenses/global-technical-support-offerings.mdx:662]:
```suggestion
                1 business day
```

File: src/content/docs/licenses/license-information/general-usage-licenses/global-technical-support-offerings.mdx [hunk 7]
```diff
@@ -574,17 +596,106 @@ These Support Plans apply only to your paid service subscription under an existi
...
 
     **Notes:**
 
     * Silver tier applies to customers with $1 to $9,999 annual spend. Gold tier applies to customers with $10,000 to $99,999 annual spend. Platinum tier applies to customers with $100,000 annual spend and above. Contact your Account Manger regarding Priority Support.
-    * Initial Support Response Time begins when the request is received by the New Relic support system.
-    * Critical means customer's business operations are severely impacted due to New Relic with no available workaround; or there is a critical security issue.
-    * This Support Plan is subject to change at any time; changes will take immediate effect.
+    * 24x7x365 support, otherwise known as "Follow the Sun" support, is only applicable to P1 business critical support cases.
+    * Initial support response time begins when the request is received by the New Relic support system.
+    * This support plan is subject to change at any time; changes will take immediate effect.
```
Comment (rhetoric101) [inline src/content/docs/licenses/license-information/general-usage-licenses/global-technical-support-offerings.mdx:687]: @homelessbirds I'm not seeing this definition about critical in this revision. Is that by design? Here's the statement: "Critical means customer's business operations are severely impacted due to New Relic with no available workaround; or there is a critical security issue.

File: src/content/docs/licenses/license-information/general-usage-licenses/global-technical-support-offerings.mdx [hunk 8]
```diff
@@ -574,17 +596,106 @@ These Support Plans apply only to your paid service subscription under an existi
...
   </Collapser>
 </CollapserGroup>
 
+## Support hours
+
+Support business hours are Monday through Friday and regionally defined as:
+
+* **AMER**: 8:00 AM – 8:00 PM Eastern Time (5:00 AM – 5:00 PM Pacific Time)
+* **APAC**: 9:00 AM – 6:00 PM India Standard Time (7:30 PM – 4:30 AM Pacific Time)
+* **EMEA**: 8:00 AM – 6:00 PM Greenwich Mean Time (12:00 AM -–10 AM Pacific Time)
```
Comment (rhetoric101) [inline src/content/docs/licenses/license-information/general-usage-licenses/global-technical-support-offerings.mdx:697]:
```suggestion
* **EMEA**: 8:00 AM – 6:00 PM Greenwich Mean Time (12:00 AM – 10 AM Pacific Time)
```

**Review by rhetoric101 — DISMISSED** [review]
@homelessbirds This looks good! I just inserted a few suggestions and questions.

File: src/content/docs/licenses/license-information/general-usage-licenses/global-technical-support-offerings.mdx (hunk 4 above)
Comment (homelessbirds) [reply src/content/docs/licenses/license-information/general-usage-licenses/global-technical-support-offerings.mdx:245]: I had discussed this with the team and we decided on keeping them numerical to match other examples of this sort of doc.

File: src/content/docs/licenses/license-information/general-usage-licenses/global-technical-support-offerings.mdx (hunk 1 above)
Comment (homelessbirds) [reply src/content/docs/licenses/license-information/general-usage-licenses/global-technical-support-offerings.mdx]: Fixed this!

File: src/content/docs/licenses/license-information/general-usage-licenses/global-technical-support-offerings.mdx (hunk 7 above)
Comment (homelessbirds) [reply src/content/docs/licenses/license-information/general-usage-licenses/global-technical-support-offerings.mdx:687]: This was removed on purpose. Thanks for flagging!

**Review by akristen — APPROVED** [review]

**Pull request merged**